### ISSUE TITLE
Forum themes shouldn't take over the existing site's style

### ISSUE THREAD

**Issue by pickhardt** [rubysherpas/forem#389]
I tried out the "twist" theme, and it applied to my existing site too.  The forum theme should just apply to the forum, in my opinion.

**Comment by parndt** [comment]
How?

**Comment by pickhardt** [comment]
The theme modified the body css (color, font, etc).  I'd prefer if every style were put inside some namespace. How you do that is up to you, but probably the easiest is to put all the HTML inside a div#forem, and then make every style in SCSS only apply to elements under the forem id.

**Comment by radar** [comment]
I suspect you're including these lines into your application stylesheets, rather than stylesheets specifically for Forem.

**Comment by pickhardt** [comment]
I just followed the instructions here: https://github.com/radar/forem/wiki/Theming

It says to apply the theme styles to your css file, so naturally I added them to application.css.  It's pretty common for add ons to namespace themselves and limit interference with the main site, so I assumed that wouldn't happen.

You guys should add a warning to the instructions or try to limit how the theme interferes with a site's style.
